src/Passes/Source/Commandline/ParameterParser.hpp

```diff
@@ -0,0 +1,105 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+namespace microsoft
+{
+namespace quantum
+{
```

Comment:
If the code uses C++17 or newer then you may want to consider 
```suggestion
namespace microsoft::quantum
{
```

Reply:
Currently, we are using C++14 as LLVM is using C++14.